src/component/short-cards/short-cards.pcss

```diff
@@ -1,14 +1,18 @@
-.short-cards {
+.short-card-link {
+  color: var(--color-black);
+  font-family: var(--font-jost);
+  font-weight: 400;
+  font-size: 16px;
+  text-decoration: none;
   display: flex;
-  flex-direction: column;
-  gap: 20px;
+  align-items: center;
+  gap: 10px;
 
-  .short-card {
-    background-color: var(--color-primary-light);
-    padding: 16px;
-    border-radius: 10px;
+  &.active {
+    color: var(--color-white);
+    font-weight: 700;
 
-    &.active {
+    .short-card {
```

Comment:
one nice thing about css custom properties compared to sass variables is that you can change just the property

for example:



```
--bg-short-card: var(--color-primary-light);
--color-short-card: var(--color-black);
.active {
  --bg-short-card: var(--color-primary);
  --color-short-card: var(--color-white);
}

.short-card { background-color: var(--bg-short-card); color: var(--color-short-card) }
```

This way you can have a single block of markup for a component, then change its values under conditions, such as `dark` or `active` or `sidebar` - you change the properties only and the component css does not need to be nested or repeated.